### DISCUSSION TITLE
Alerts on some thresholds (ntfy as receiver)

### DISCUSSION THREAD

**Discussion by flowcool** [ghostfolio/ghostfolio#3362] [Ideas]
Hello Team,

I was wondering if at some point, it would be feasible to put some alerts when a/few thresholds are reached ?
Goal is not to have a real time alert mechanism; but if a trend is there, it might help to take decision.

Ntfy is a good platform to receive alerts.

**Comment by gmag11** [comment]
Hi, I totally agree. It may be implemented by configuring an user callback. In that way alert output would be platform agnostic (Gotify, Ntfy, etc, or even direct to Telegram)
Some template system should be added to build notification messages.
What do you think?

**Comment by gmag11** [comment]
@dtslvr How often is market data retrieved? Do you think an alert system would be feasible and useful with current data providers?
